yaksh/static/yaksh/js/requesthandler.js

```diff
@@ -113,14 +115,17 @@ $(document).ready(function(){
   // Setting code editors initial content
   global_editor.editor.setValue(init_val);
 
-  function reset_editor() {
+  $('#code').submit(function(e) {
+    lock_screen();
+    var data = $(this).serializeArray();
+    ajax_check_code($(this).attr("action"), "POST", "html", data, null)
+        e.preventDefault(); // To stop the default form submission.
+  });
+
+  reset_editor = function() {
```

Comment:
This will always reset to the init value; which can also be the last submitted code. Are we sure that is what we want? Well from the name clear text area, I was expecting it would clear everything except the snippet.

Reply:
Yes, the init value is last submitted code if any!
I think reset should simply display snippet on click or blank if no snippet.

Reply:
Will rename button to 'Undo Changes'